tests/cli_test_utils.py

```diff
@@ -0,0 +1,36 @@
+import os, sys
+import subprocess
+
+def run_cli(cli_entry, *args):
```

Comment:
Is there any reason not to put `CLI_ENTRY`
```
CLI_ENTRY = [
        sys.executable,
        "-m",
        "agentstack.main",
    ]
    ```
Since we aren't varying the entry anywhere, this would add a lot to the readability of tests even if we have a reason to undo it later.

Reply:
Totally agree, will add that.